### DISCUSSION TITLE
Buildkite dependency resolution

### DISCUSSION THREAD

**Discussion by peter-svensson** [renovatebot/renovate#25058] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.4.0

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

If a plugin is has a tags `v1.0.0`, `v1.0.1` and `v1`, where `v1` points to the same commit as `v1.0.1` I would expect that a plugin declaration as follows:

`https://bitbucket.org/some-org/some-buildkite-plugin#v1` 

would not trigger a dependency update to `https://bitbucket.org/some-org/some-buildkite-plugin#v1.0.1` since it is the "same" version?

The same would be said for Github Actions I guess, where moving the major version is a common way to "update" the actual action being triggered without having to manage the explicit version changes. Not sure if this is true for Github Actions or not.

Could be that this is the expected/wanted behavior but it will lead to a lot of "unnecessary" Pull Requests I guess or can it be configured somewhere that renovate ignores minor upgrades perhaps?

### What you tried so far.

See "wanted end result"

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
I'm guessing it's using semver-coerced versioning, which will do a best effort to parse and sort everything which is version-ish. If you switch to Docker versioning, it will retain precision by default

**Comment by peter-svensson** [comment, marked as answer]
Thanks!
Did some testing and this seems to "work":

```
packageRules: [
  {
      matchManagers: ["buildkite"],
      matchUpdateTypes: ["minor", "patch"],
      enabled: false
  }
```

This will just trigger updates on major versions. will be ok for my usecase atm.
